packages/fxa-payments-server/public/locales/en-US/unused-strings.ftl

```diff
@@ -0,0 +1,80 @@
+### UNUSED STRINGS?
```

Comment:
Could not locate where these strings are used. I'm not sure if these strings are deprecated.

This ftl file is not currently concatenated into the l10n package.

Reply:
@vpomerleau are there still unused strings? I don't see any strings commented out at the bottom of `src/ftl`

Reply:
Ah never mind. I saw your conversation with @sardesam. Apologies for the double ping.